singlestoredb/fusion/handlers/job.py

```diff
@@ -0,0 +1,594 @@
+
+        if params['extended']:
+            res.add_field('Description', result.STRING)
+            res.add_field('TerminatedAt', result.DATETIME)
+            res.add_field('CreateSnapshot', result.BOOL)
+            res.add_field('MaxDurationInMins', result.INTEGER)
+            res.add_field('ExecutionIntervalInMins', result.INTEGER)
+            res.add_field('Mode', result.STRING)
+            res.add_field('StartAt', result.DATETIME)
+            res.add_field('ResumeTarget', result.BOOL)
+
+            def fields(job: Any) -> Any:
+                database_name = None
+                resume_target = None
+                target_id = None
+                target_type = None
+                if job.target_config is not None:
+                    database_name = job.target_config.database_name
+                    resume_target = job.target_config.resume_target
+                    target_id = job.target_config.target_id
+                    target_type = job.target_config.target_type.value
+                return (
+                    job.job_id,
+                    job.name,
+                    dt_isoformat(job.created_at),
+                    job.enqueued_by,
+                    job.completed_executions_count,
+                    job.execution_config.notebook_path,
+                    database_name,
+                    target_id,
+                    target_type,
+                    job.description,
+                    dt_isoformat(job.terminated_at),
+                    job.execution_config.create_snapshot,
+                    job.execution_config.max_duration_in_mins,
+                    job.schedule.execution_interval_in_minutes,
+                    job.schedule.mode.value,
+                    dt_isoformat(job.schedule.start_at),
+                    resume_target,
+                )
+        else:
+            def fields(job: Any) -> Any:
+                database_name = None
+                target_id = None
+                target_type = None
+                if job.target_config is not None:
+                    database_name = job.target_config.database_name
+                    target_id = job.target_config.target_id
+                    target_type = job.target_config.target_type.value
+                return (
+                    job.job_id,
+                    job.name,
+                    dt_isoformat(job.created_at),
+                    job.enqueued_by,
+                    job.completed_executions_count,
+                    job.execution_config.notebook_path,
+                    database_name,
+                    target_id,
+                    target_type,
+                )
+
+        res.set_rows([fields(job) for job in jobs])
+
+        if params['like']:
+            res = res.like(Name=params['like'])
+
+        return res
+
+
+ShowJobsHandler.register(overwrite=True)
+
+
+class ShowJobExecutionsHandler(SQLHandler):
+    """
+    SHOW JOB EXECUTIONS job_id
```

Comment:
For this one, I think it would be more confusing to return the `executions` for multiple jobs. The same applies for the new one I added that returns the `parameters` for a job.